tortoise/contrib/test/__init__.py

```diff
@@ -179,6 +179,8 @@ def _init_loop(self) -> None:
         pass
 
     async def _setUp(self) -> None:
+
+
```

Comment:
black is gonna complain about the double newline here :-P

And I was just about to ask you to put the async back...

Reply:
yeah, messed that one up during rebasing, sorry